src/frontend/helpers/library.ts

```diff
@@ -157,6 +160,9 @@ const launch = async ({
             {
               text: t('box.no'),
               onClick: async () => {
+                if (runner === 'hyperplay') {
+                  return
+                }
```

Comment:
Users might think this is a bug when clicking "No" in response to "This game has an update, apply update now?" after requesting to launch the game doesn't launch the game.

Could we instead not include the "No" button in the buttons array if runner === 'hyperplay' instead of early return so there is only a yes?

Open to other suggestions as well

Reply:
I dont understand. If you say no it is because you don't want to update the game and you also won't play it until you update.

I think it is even more confusing to have only the yes button.

Reply:
Added another dialog that says that it is not possible to play the game without updating. 
What do you think?

Reply:
yeah that works 👌